src/components/com_tjcertificate/site/models/forms/externalcertificate.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="utf-8"?>
+<form>
+	<fieldset>
+		<field
+			name="id"
+			type="hidden"
+			default="0"
+			label="COM_TJCERTIFICATE_FORM_LBL_EVENT_ID"
+			readonly="true"
+			class="readonly"
+			description="JGLOBAL_FIELD_ID_DESC" />
+
+		<field
+			name="user_id"
+			type="hidden" />
+
+		<field
+			name="assigned_user_id"
+			type="sql"
+			label="COM_TJCERTIFICATE_FORM_LBL_ASSIGNED_USER"
+			description="COM_TJCERTIFICATE_FORM_LBL_ASSIGNED_USER_DESC"
+			query="SELECT id, name FROM #__users where block = 0"
+			key_field="id"
+			value_field="name"
+			>
+			<option value="">COM_TJCERTIFICATE_CERTIFICATE_USER_SELECT_OPTION</option>
```

Comment:
Why we need this field?

Reply:
to add record on behalf of user, who have manage access will able to add record for user.